src/components/FilterControl.jsx

```diff
@@ -0,0 +1,69 @@
+import { FaShoppingBasket, FaSpa, FaWineBottle } from "react-icons/fa";
+import { GiCampingTent } from "react-icons/gi";
+import { MdCasino, MdLocalBar, MdLocalCafe, MdOutlineRestaurant, MdSelectAll } from "react-icons/md";
+import { TbHotelService } from "react-icons/tb";
+import styles from "../css/map-list-opener.module.css";
+import { FaFilter } from "react-icons/fa6";
+import { useState, useCallback } from "react";
+
+function FilterControl({ language, filterType, setFilterType }) {
+    const [filterOpen, setFilterOpen] = useState(false);
+    const size = 16;
+    const options = [
+        { type: "mind", category: "all", category_de: "Alles", icon: <MdSelectAll size={size} /> },
+        { type: "bár", category: "bar", category_de: "Bar", icon: <MdLocalBar size={size} /> },
+        { type: "borászat", category: "winery", category_de: "Weingut", icon: <FaWineBottle size={size} /> },
+        { type: "étterem", category: "restaurant", category_de: "Restaurant", icon: <MdOutlineRestaurant size={size} /> },
+        { type: "kávézó", category: "café", category_de: "Cafe", icon: <MdLocalCafe size={size} /> },
+        { type: "spa&sport", category: "spa&sport", category_de: "Spa und Sport", icon: <FaSpa size={size} /> },
```

Comment:
Ezt nem szokozokkel kene irni? 🤔 `spa & sport`?

Reply:
@pisipite írta így, ha leokézza én átírom, de akkor a táblázatban is javítani kell.